backends/ebpf/psa/ebpfPsaGen.cpp

```diff
@@ -690,6 +690,31 @@ bool ConvertToEBPFControlPSA::preorder(const IR::ExternBlock* instance) {
     return false;
 }
 
+class IndexVisitor : public Inspector {
```

Comment:
I would give a more precise name to this visitor, like "PathOnLHS"; it really detects whether there is an assignment to a specific path expression.